worlds/tits_the_3rd/client/client.py

```diff
@@ -100,13 +229,59 @@ async def tits_the_3rd_watcher(ctx: TitsThe3rdContext):
             continue
 
         if ctx.game_interface.should_write_world_player_identifier():
+            logger.info(f"Game Start Dectected. Setting up AP Verification Hook")
             ctx.game_interface.write_world_player_identifier(ctx.world_player_identifier)
+            logger.info(f"AP Verification Hook Set. You can now save and resume the game for this AP Seed")
             continue
 
+        try:
+            if ctx.exit_event.is_set():
+                break
+            if not ctx.server:
+                continue
+            if ctx.game_interface.should_send_and_recieve_items(ctx.world_player_identifier):
```

Comment:
Everything past this point should be under `if ctx.game_interface.should_send_and_recieve_items`. Lets just have one if statement.

... Or I was going to say that, but I guess this is to prevent the case where the user disconnects from their save halfway into this loop?

Reply:
Probably not needed, but it's probably safe to have just in case